Resources/Prototypes/_Carpmosia/Maps/feint.yml

```
@@ -0,0 +1,65 @@
+- type: gameMap
+  id: Feint
+  mapName: 'Feint'
+  mapPath: /Maps/_Carpmosia/feint.yml
+  minPlayers: 10
+  maxPlayers: 35 
+  stations:
+    Feint:
+      stationProto: StandardNanotrasenStation
+      components:
+      - type: StationNameSetup
+        mapNameTemplate: '{0} Feint Station {1}'
+        nameGenerator:
+          !type:NanotrasenNameGenerator
+          prefixCreator: '-TE'
+      - type: StationEmergencyShuttle
+        emergencyShuttlePath: /Maps/_Carpmosia/Shuttles/emergency_feint.yml
+      - type: StationJobs
+        availableJobs: # 44 jobs total w/o latejoins & interns, 59 jobs total w/ latejoins & interns
```

```suggestion
        availableJobs: # 43 jobs total w/o latejoins & interns, 58 jobs total w/ latejoins & interns
```
No BSO on carpmos so off by one.

---

Is it incorrect though? I counted them

---

I counted like 3 times last night to make sure and if I somehow fucked it up I will (remembers suicide jokes are neither healthy for myself nor those around me) try to limit the actions i take late at night when I'm not at my desktop.

---

I think the total count here is right but the service count is wrong

---

yeah, we were both wrong :>